include/modem/nrf_modem_lib.h

```diff
@@ -113,7 +113,7 @@ struct nrf_modem_lib_shutdown_cb {
  * When nrf_modem_lib_shutdown() is called a thread can call this function to be
  * woken up next time nrf_modem_lib_init() is called.
  */
-void nrf_modem_lib_shutdown_wait(void);
+__deprecated void nrf_modem_lib_shutdown_wait(void);
```

Comment:
I might be missing something here, but looks like the changelog comment on `nrf_modem_lib_get_init_ret` doesn't match the function marked as deprecated, `nrf_modem_lib_shutdown_wait`

Reply:
I plan to deprecate that function too btw, but I'll will be a separate PR.